core/db/curibio/alembic/versions/1fbfdf12dfbb_add_maunits.py

```diff
@@ -0,0 +1,51 @@
+"""add MAUnits
+
+Revision ID: 1fbfdf12dfbb
+Revises: 258ca806ee8b
+Create Date: 2022-03-16 21:42:22.132660
+
+"""
+import os
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '1fbfdf12dfbb'
+down_revision = '258ca806ee8b'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute("CREATE USER curibio_mantarray_ro")
+
+    table = op.create_table(
+        "maunits",
+        sa.Column("serial_number", sa.String(12), primary_key=True),
+        sa.Column("hw_version", sa.String(12), nullable=False),
+    )
+
+    op.bulk_insert(table,
+        [
+            {"serial_number": "M02022055001", "hw_version": "2.2.0"},
+            {"serial_number": "M02022055002", "hw_version": "2.2.0"},
+            {"serial_number": "M02022055003", "hw_version": "2.2.0"},
+            {"serial_number": "M02022055004", "hw_version": "2.2.0"},
+        ],
+    )
+
+    op.execute("GRANT SELECT ON maunits TO curibio_mantarray_ro")
+
+    mantarray_ro_pass = os.getenv("MANTARRAY_RO_PASS")
```

Comment:
is this using the sealed secret "curibio_mantarray_ro" in the curibio-mantarray-creds.yaml?

Reply:
it will in the CI workflow, I have a .env file I use locally to load all the needed passwords, I think the plan is to run all the migrations from CI. For the test cluster you can read out the secrets with kubectl and put them in a .env if you want to test out migrations on the test db